tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -20,6 +20,7 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
 
 *** Variables ***
 ${yml}  version: "2"\nservices:\n${SPACE}web:\n${SPACE}${SPACE}image: python:2.7\n${SPACE}${SPACE}ports:\n${SPACE}${SPACE}- "5000:5000"\n${SPACE}${SPACE}depends_on:\n${SPACE}${SPACE}- redis\n${SPACE}redis:\n${SPACE}${SPACE}image: redis\n${SPACE}${SPACE}ports:\n${SPACE}${SPACE}- "5001:5001"
+${link-yml}  version: "2"\nservices:\n${SPACE}redis:\n${SPACE}${SPACE}image: redis:alpine\n${SPACE}${SPACE}container_name: redis\n${SPACE}${SPACE}ports: ["6379"]\n${SPACE}web:\n${SPACE}${SPACE}image: busybox\n${SPACE}${SPACE}container_name: a.b.c\n${SPACE}${SPACE}links:\n${SPACE}${SPACE}- redis:aaa\n${SPACE}${SPACE}command: ["ping", "aaa"]
```

Comment:
If you use the same name for redis as the first yml, you should call docker-compose down on the first compose app before you bring this one up.  Until we have docker rename, you can't bring up multiple apps that have the same name for services.